### PR TITLE
Hauki 378 improve rule frequency field

### PR DESCRIPTION
**Pull request by minevala**
- Add default value for rule's frequency "Jokainen"
- Disable rule's start field if 'Pariton' or 'Parillinen' frequency is selected

Näyttökuva 2021-2-11 kello 14 50 37

### CODE REVIEW

File: src/opening-period/page/EditOpeningPeriodPage.test.tsx [hunk 1]
```diff
@@ -687,4 +658,76 @@ describe(`<EditNewOpeningPeriodPage />`, () => {
       expect(mockHistoryPush).toHaveBeenCalledWith(testResourcePageUrl);
     });
   });
+
+  it('should disable rule start when "Parillinen" frequency is selected', async () => {
```
Comment (VilleMiekkoja) [inline src/opening-period/page/EditOpeningPeriodPage.test.tsx:662]: It seems we have multiple places where we could have this test, and I don't really have a strong opinion which place would be best:

1. EditNewOpeningPeriodPage
2. CreateNewOpeningPeriodPage
3. OpeningPeriodForm
4. Rule.test.tsx

What do you think which place should have which tests?

File: src/opening-period/rule/Rule.tsx [hunk 2]
```diff
@@ -151,51 +152,50 @@ export default function Rule({
...
-      selectedFrequency?.value.frequency_modifier || null
-    );
+    setValue(frequencyModifierField, selectedModifier || null);
     setValue(
       frequencyOrdinalField,
       selectedFrequency?.value.frequency_ordinal || null
     );
-  };
+
+    if (selectedModifier) {
```
Comment (VilleMiekkoja) [inline src/opening-period/rule/Rule.tsx:185]: Does this mean it is either 'pariton' or 'parillinen'?

File: src/opening-period/page/EditOpeningPeriodPage.test.tsx (hunk 1 above)
Comment (minevala) [reply src/opening-period/page/EditOpeningPeriodPage.test.tsx:662]: I could move rule test under OpeningPeriodForm or Rule.test.tsx? 🤔

File: src/opening-period/rule/Rule.tsx (hunk 2 above)
Comment (minevala) [reply src/opening-period/rule/Rule.tsx:185]: Yes, start does not apply anything to the rule when it has a modifier. I could add a constant which would make easier to read tough 🤔

Comment (minevala) [reply src/opening-period/rule/Rule.tsx:185]: I added a comment instead :grin

**Review by VilleMiekkoja — APPROVED** [review]
Good job 🎖️

**Pull request merged**